README.md

```diff
@@ -55,6 +55,24 @@ and its upstream project, [Open Data Hub](https://opendatahub.io/).
    * The wrapper script has several arguments and you can find details in the dedicated document file. See [run_args.md](ods_ci/docs/RUN_ARGUMENTS.md)
    * As alternative, you can run any of the test cases by creating the python virual environment, install the packages in [poetry.lock](poetry.lock) and running the `robot` command directly
 
+# Test tiers
+  - Smoke
+    - Tests executed as part of productization pipeline to verify basic RHOAI functionality
+    - Tests should cover only critical functionality, verifying that build is usable and can be accepted for testing
+    - Tests should be ideally 100% stable and reliable
+    - Execution time of every single test should be less than 1 minute
```

Comment:
I'm not sure about 1 minute timeout. It is often problem to create a pod because of downloading images. I would suggest:
smoke - 3 minutes
sanity - 10 minutes
tier1 - 20 minutes

Reply:
Pod downloading could be put out of the test, and have one "setup test" that gets some image set prefetched, that the subsequent tests can then use... Limiting the size of this image set (not extending it without a good reason) would also help to make things faster.

Maybe specify the limits for first run and for subsequent runs of the test. I do like the idea of "1 minute test" and would like to preserve it in some form. I was often running a test multiple times recently, when debugging fails.